package.json

```diff
@@ -1,60 +1,67 @@
 {
   "name": "vyapi",
```

Comment:
do you realize that there are lot of changes in this file. r u sure it's correct